sdk/keyvault/Azure.Security.KeyVault.Administration/api/Azure.Security.KeyVault.Administration.netstandard2.0.cs

```diff
@@ -26,6 +52,30 @@ public enum ServiceVersion
             V7_2_Preview = 1,
         }
     }
+    public partial class KeyVaultBackupRestoreClient
+    {
+        protected KeyVaultBackupRestoreClient() { }
+        public KeyVaultBackupRestoreClient(System.Uri vaultUri, Azure.Core.TokenCredential credential) { }
+        public KeyVaultBackupRestoreClient(System.Uri vaultUri, Azure.Core.TokenCredential credential, Azure.Security.KeyVault.Administration.KeyVaultBackupRestoreClientOptions options) { }
+        public virtual System.Uri VaultUri { get { throw null; } }
+        public virtual Azure.Response<Azure.Security.KeyVault.Administration.Models.FullBackupDetails> GetFullBackupDetails(string jobId, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
These methods should be hidden and replaced by an operation resume ctor.

`new FullBackupOperation(jobId, KeyVaultBackupRestoreClient)`